dependencies/xerces-c/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ target_include_directories(${PROJECT_NAME} SYSTEM INTERFACE
 )
 
 target_link_libraries(${PROJECT_NAME} INTERFACE
-  ${INSTALL_DIR}/${CMAKE_INSTALL_LIBDIR}/${CMAKE_STATIC_LIBRARY_PREFIX}${LIBRARY_NAME}${CMAKE_STATIC_LIBRARY_SUFFIX}
+  ${INSTALL_DIR}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}${LIBRARY_NAME}${CMAKE_STATIC_LIBRARY_SUFFIX}
```

Comment:
As I feared, this breaks on RHEL.

I think the easiest work around right now is to have the CI set the `CMAKE_INSTALL_PREFIX` to something not `/usr` or `/usr/local` and leave this file alone.